### ISSUE TITLE
Enhancement: Additional Cuckoo module network support

### ISSUE THREAD

**Issue by mtonsmann** [VirusTotal/yara#1175]
See TL;DR

As an avid VT user, I was excited to learn about support for hunting with yara, including support for the cuckoo module. When I started to use the module, I noticed that it did not yet support many of the things that are available in many cuckoo JSON reports.

When looking through issues and PRs, I found #958 which wanted to update the module to support the latest cuckoo JSON format. Unfortunately it looks like this had some issues that were never fixed and it was not merged.

While updating the module to support the latest JSON as well as keeping it backwards compatible seems worthwhile, my request is to simply support 3 new functions to parse existing functionality from the cuckoo reports:

network.host(regexp)
	example: cuckoo.network.host(/192\.168\.1\.1/)
network.tcp(regexp)
	example: cuckoo.network.tcp(/192\.168\.1\.1:80/)
network.udp(regexp)
	example: cuckoo.network.tcp(/192\.168\.1\.1:53/)

I have read the source and think I can add these methods via a PR. I am also happy to write the documentation to support these methods. 

As far as I can tell there is no need to support new versions of cuckoo reports, but if there is interest then I could add support for those as well.

TL;DR before I start work on a PR, are the maintainers interested in seeing support for additional network methods in the cuckoo module?

**Comment by mtonsmann** [comment]
added in #1177
